src/ucs/config/parser.c

```diff
@@ -1238,26 +1238,48 @@ static void ucs_config_parser_mark_env_var_used(const char *name, int *added)
     pthread_mutex_unlock(&ucs_config_parser_env_vars_hash_lock);
 }
 
-static int ucs_config_parse_config_file_line(void *user, const char *section,
-                                             const char *name,
-                                             const char *value)
+static khiter_t ucs_config_get_iter_from_config_file(const char *name)
 {
-    int result;
-    khiter_t iter;
+    return kh_get(ucs_config_map, &ucs_config_file_vars, name);
+}
 
-    iter = kh_put(ucs_config_map, &ucs_config_file_vars,
-                  ucs_strdup(name, "parse_config_file_line name"), &result);
+static char *ucs_config_get_value_from_config_file(const char *name)
```

Comment:
do we really need this wrapper?

Reply:
I think so. It allows for a cleaner code in `ucs_config_apply_env_vars` (`env_value = ucs_config_get_value_from_config_file(buf);`), instead of dealing with the hash table right there.